Dockerfile

```diff
@@ -1,7 +1,12 @@
 FROM golang:1.4
 
+RUN apt-get update && \
+    apt-get install -y librados-dev && \
+    rm -rf /var/lib/apt/lists/*
+
 ENV DISTRIBUTION_DIR /go/src/github.com/docker/distribution
 ENV GOPATH $DISTRIBUTION_DIR/Godeps/_workspace:$GOPATH
+ENV DOCKER_BUILDTAGS include_rados
```

Comment:
The Makefile is used by the docker build file, so defining them there would probably not be correct. Ideally, we'd like to maintain responsibility with levels (makefile: what and how to build, dockerfile: environment for build, script: kick off different kinds of builds). I suspect we will have a build script to manage these. For now, this is okay, since we don't have higher-level construct.